ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -1,8 +1,131 @@
 # Release Notes For Ansible AVD 3.x.x
 
+Documentation for AVD version `3.5.0` [available here](https://www.avd.sh/en/v3.5.0/)
```

Comment:
we have a catch-22 here. We cannot create this URL without the release, and we cannot create the release without RN pointing to this url.

I suggest to change the 3.5 to point to `/en/stable` and then we change to v3.5.0 when we create v3.6.0